src/DeserializationPojo/SingleResourcePojo.java

```diff
@@ -0,0 +1,22 @@
+package DeserializationPojo;
+
+public class SingleResourcePojo {
```

Comment:
There is a convention where when you are getting a single thing you say Get Thing and when you are getting multiple things you say List Things. So you can use that convention to name this class better as `GetResourseResponsePojo`.